lib/hanami/middleware/body_parser/class_interface.rb

```diff
@@ -9,15 +9,21 @@ class BodyParser
       # @api private
       # @since 1.3.0
       module ClassInterface
+        # @api private
+        # @since 2.0.0
+        def new(app, parser_specs)
```

Comment:
Would you mind leaving some doc comments here about the expected structures for the parser_specs?

Reply:
@timriley done